### PR TITLE
Bootstrap module

### PR DESCRIPTION
**Pull request by piuccio**
Remove some duplication between `collections` and `config` by moving the loading logic into it's own module.

Moreove the frontend configuration is taken from an endpoint instead of being injected in the markup.

### CODE REVIEW

**Comment by johnduffell** [comment]
oh man, I'd love to approve this but I need screenshots :P

File: facia-tool/app/controllers/FrontendDependentController.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        Play.isDev,
+        Configuration.environment.stage,
+        Edition.all.map(_.id.toLowerCase),
+        FaciaToolConfiguration.sectionsFromNav,
+        request.user.email,
+        request.user.avatarUrl,
+        Configuration.faciatool.adminPressJobLowPushRateInMinutes,
+        Configuration.faciatool.adminPressJobHighPushRateInMinutes,
+        Configuration.faciatool.adminPressJobStandardPushRateInMinutes,
+        "https://4527e03d554a4962ae99a7481e9278ff@app.getsentry.com/35467",
```
Comment (johnduffell) [inline facia-tool/app/controllers/FrontendDependentController.scala]: is this id something that should be in the config?

File: facia-tool/app/controllers/FrontendDependentController.scala [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  avatarUrl: Option[String],
+  lowFrequency: Int,
+  highFrequency: Int,
+  standardFrequency: Int,
+  sentryPublicDSN: Option[String],
+  fixedContainers: Seq[ContainerJsonConfig],
+  dynamicContainers: Seq[ContainerJsonConfig]
+)
+
+object FrontendDependentController extends Controller {
```
Comment (johnduffell) [inline facia-tool/app/controllers/FrontendDependentController.scala]: not 100% sure the naming of this class is right? should be FrontsConfigController maybe?

Comment (piuccio) [inline facia-tool/app/controllers/FrontendDependentController.scala]: I want to make clear that this class depends on `frontend`. Eventually it'll have to be removed when we split dependencies.

Comment (johnduffell) [inline facia-tool/app/controllers/FrontendDependentController.scala]: I understand now - also may be worth putting that informration into a comment just so everyone knows who looks later

**Comment by johnduffell** [comment]
this is big! but it looks ok especially the scala :+1: ;)

**Pull request merged**